cfbs/commands.py

```diff
@@ -1000,3 +1001,103 @@ def input_command(args, input_from="cfbs input"):
         files_to_commit.append(input_path)
     config.save()
     return Result(0, do_commit, None, files_to_commit)
+
+
+def set_input_command(name, src_path):
+    config = CFBSConfig.get_instance()
+    module = config.get_module_from_build(name)
+    if module is None:
+        log.error("Module '%s' not found" % name)
+        return 1
+
+    spec = module.get("input")
+    if spec is None:
+        log.error("Module '%s' does not accept input" % name)
+        return 1
+    log.debug("Input spec for module '%s': %s" % (name, pretty(spec)))
+
+    if src_path is None:
+        try:
+            data = json.load(stdin)
```

Comment:
Use `OrderedDict` explicitly so python 3.5 also has predictable dictionaries:

```suggestion
            data = json.load(stdin, object_pairs_hook=OrderedDict)
```